apps/condo/domains/ticket/utils/handlers.js

```diff
@@ -113,12 +136,86 @@ const handleTicketEvents = async (requestData) => {
     }
 
     if (eventTypes[STATUS_CHANGED_EVENT_TYPE]) {
-        // TODO(DOMA-2434): Add logic for sending notifications to resident on ticket status change
+        let type
+        switch (nextStatusId) {
+            case STATUS_IDS.OPEN:
+                if (statusReopenedCounter > 0)
+                    type = updatedBy !== client && TICKET_STATUS_RETURNED_TYPE
+                else
+                    type = createdBy !== client && TICKET_STATUS_OPENED_TYPE
+                break
+
+            case STATUS_IDS.IN_PROGRESS:
+                type = TICKET_STATUS_IN_PROGRESS_TYPE
+                break
+
+            case STATUS_IDS.COMPLETED:
+                type = TICKET_STATUS_COMPLETED_TYPE
+                break
+
+            case STATUS_IDS.DECLINED:
+                type = TICKET_STATUS_DECLINED_TYPE
+                break
+        }
+
+        if (type) {
+            await sendMessage(context, {
+                lang,
+                to: { user: { id: client } },
+                type,
+                meta: {
+                    dv: 1,
+                    data: {
+                        ticketId: updatedItem.id,
+                        ticketNumber: updatedItem.number,
+                        userId: client,
+                        domain: TICKET_DOMAIN,
```

Comment:
why do you need a domain?

Reply:
`comment` is not a domain name

Reply:
> why do you need a domain?

mobile app need info on domain where notification happened for routing sake